utils/algorithms.hpp

```diff
@@ -15,4 +16,5 @@ int compareId(const Shape*, const Shape*);
 int compareArea(const Shape*, const Shape*);
 int comparePerimeter(const Shape*, const Shape*);
 void sort(Vector<Shape*>&, Comparator = compareId);
-}
\ No newline at end of file
+void getTo(QTextStream&, const QChar&);
+}
```

Comment:
End the file with an empty line
```suggestion
}

```